### ISSUE TITLE
job-offer

### ISSUE THREAD

**Issue by roodrallec** [Lissy93/happy-app#3]
@Lissy93 love this app and your approach to coding. I'm opening this issue to ask if you'd be interested in working to solve problems in the mental health space with us, [Syndi](https://www.syndi.health/).

At Syndi our mission is to build the world’s most accessible healthcare monitoring platform, that gives people the power to care for themselves. We were founded in November 2020 at [Entrepreneur First](https://www.joinef.com/) and have now received funding from them to begin building a team of awesome, passionate devs who can tackle this problem with us and help shape our product for the upcoming pilots and launch. We’re looking for people to help us tackle the following kinds of problems:

- Someone’s step count has suggested they are no longer leaving the house, is this due to COVID or is this because they are going through an episode of depression?

- A user has said that they are looking for apps that can improve their mental wellbeing, which apps are actually helping them and which are actually worsening the state of their depression or anxiety?

While Syndi is a young company, you’ll be part of our core team, help build a culture that developers around the world hear about, and be integrated with our experienced supporting group of advisors, including a [co-founder of the Raspberry Pi](https://en.wikipedia.org/wiki/Alan_Mycroft). 

TLDR;

If you're currently looking for new opportunities and interested in tackling problems in the mental health space through coding, reach out to us at:

[dev+application@syndi.health](mailto:dev+application@syndi.health)

<img src="https://syndi-public-assets.s3.eu-west-2.amazonaws.com/Syndi-logo.png" width="120" height="120">
